qiling/os/macos/syscall.py

```diff
@@ -338,14 +339,14 @@ def ql_syscall_mmap2_macos(ql, mmap2_addr, mmap2_length, mmap2_prot, mmap2_flags
 
 # 0xca
 def ql_syscall_sysctl(ql, name, namelen, old, oldlenp, new_arg, newlen):
-    ql.nprint("sysctl(name:0x{:X}, namelen:0x{:X}, old:0x{:X}, oldlenp:0x{:X}, new:0x{:X}, newlen:0x{:X})".format(
+    ql.nprint("sysctl(name: 0x{:X}, namelen: 0x{:X}, old: 0x{:X}, oldlenp: 0x{:X}, new: 0x{:X}, newlen: 0x{:X})".format(
```

Comment:
all these we should change to format string output, 3.5 compatible printout.

Reply:
sure

Reply:
lots of them tho, this can be next PR. if u agree